### PR TITLE
Record message and session credits on transactions

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Depends on #1077, otherwise ready for review.

File: go/billing/api.py [hunk 1]
```diff
@@ -664,10 +673,10 @@ def get_transaction_list(self, account_number, page_number,
...
             SELECT id, account_number, message_id,
                    tag_pool_name, tag_name,
-                   message_direction, message_cost,
-                   session_created, session_cost,
-                   markup_percent, credit_factor, credit_amount,
-                   status, created, last_modified
+                   message_direction, message_cost, storage_cost,
+                   session_created, session_cost, markup_percent,
+                   message_credits, storage_credits, session_credits,
+                   credit_factor, credit_amount, status,created, last_modified
```
Comment (hodgestar) [inline go/billing/api.py]: Missing comma in `status,created`.

File: go/billing/models.py [hunk 2]
```diff
@@ -174,6 +187,12 @@ def message_credit_cost(self):
             self.message_cost, self.markup_percent)
 
     @property
+    def storage_credit_cost(self):
+        """Return the calculated cost per session (in credits)."""
+        return self.calculate_storage_credit_cost(
+            self.session_cost, self.markup_percent)
```
Comment (hodgestar) [inline go/billing/models.py]: Needs the updated #1077 merged in.

**Comment by hodgestar** [comment]
Two very minor comments, looks good otherwise.

File: go/billing/api.py (hunk 1 above)
Comment (justinvdm) [inline go/billing/api.py]: Comma or space?

Comment (hodgestar) [inline go/billing/api.py]: Oops. Space. :)

**Comment by justinvdm** [comment]
Ready for re-review.

**Comment by hodgestar** [comment]
:+1:

Ready to land when we land #1077.

**Comment by justinvdm** [comment]
#1118 includes this PR's changes, closing.

**Pull request closed without merging**